TheThingsNetwork/lorawan-stack
# Support NbTrans controls while using Dynamic ADR mode

#### Summary



Closes #6966 

#### Changes



- Add store for band definitions
- Add NbTrans override fields (default override and specific override)
- Add encoding and decoding of the checkbox
- Fix browser console errors

#### Testing



https://github.com/TheThingsNetwork/lorawan-stack/assets/56658938/3defb1ed-9673-4c1b-bcf6-2f9cccd7c90e

1. Login into the console
2. Go to any device/create and application and device
3. Go to device settings, to Network Server section and open the collapsed "Avanced MAC settings" section.
4. Select ADR Dynamic mode.
5. Uncheck `Use default settings for number of retransmissions`
6. Click `Add data rate specific override` and add the data rate and NbTrans
7. Then save changes. The specific override should be saved. 
8. Remove the specific override and refresh.
9. There should be no overrides.


#### Checklist
